src/core/egg-herbie.rkt

```diff
@@ -1265,7 +1312,7 @@
 ;;     - scheduler: `(scheduler . <name>)` [default: backoff]
 ;;        - `simple`: run all rules without banning
 ;;        - `backoff`: ban rules if the fire too much
-(define (make-egg-runner exprs reprs schedule #:context [ctx (*context*)])
+(define (make-egg-runner batch roots reprs schedule #:context [ctx (*context*)])
```

Comment:
`make-egg-runner` takes now `batch` and `roots`, therefore a lot of occurrences of `make-egg-runner` has been changed below